setup.py

```diff
@@ -0,0 +1,13 @@
+import sys
+import subprocess
+
+# implement conda as a subprocess:
+
+subprocess.check_call([sys.executable, '-m', 'pip', 'install',
+                       'python3-saml'])
+
+subprocess.check_call([sys.executable, '-m', 'pip', 'install',
+                       '--force-reinstall', '--use-pep517', 'lxml', 'lxml'])

```

Comment:
heroku runs setup.py automatically on build, so I am using this to enforce installation order before requirements.txt is installed.

Reply:
@simranmakandar98 Ok.